src/components/CodeEditor/CodeEditor.js

```diff
@@ -59,7 +59,10 @@ class CodeEditor extends Component {
           O Babel não pôde ser carregado.
```

Comment:
are this file and the below added intentionally in this change?

Reply:
No! I just modified the refs-and-the-dom.md file, and didn't created or modified any other file!

But, I executed the `yarn check-all`, and maybe this file was created after this.

Reply:
ok hopefully upstream does the same, please ignore then. resolving